R/plotPercentiles.R

```diff
@@ -0,0 +1,107 @@
+#' Comparison plots show user selected variables obtained from different scenario
```

Comment:
This is not a good description of the function, as it basically applies to any plot function in this library.

Reply:
Fixed